rhoas-client/src/test/java/com/redhat/service/bridge/rhoas/RhoasAPI.java

```diff
@@ -0,0 +1,29 @@
+package com.redhat.service.bridge.rhoas;
+
+import java.time.Duration;
+
+import javax.inject.Inject;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import com.redhat.service.bridge.rhoas.dto.TopicAndServiceAccountRequest;
+import com.redhat.service.bridge.rhoas.dto.TopicAndServiceAccountResponse;
+
+@Path("/rhoas/")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+public class RhoasAPI {
```

Comment:
Add a comment to explain the reason of this test class. As far as I remember you told me it is necessary to test that there is no blocking code in the I/O Thread of Vertx. Am I right?

Reply:
I wrote this dummy test API to simulate a real usage of the `RhoasClient` in a real JAX-RS bean.
This allows to spot errors caused by the client mistakenly blocking the Vert.x event loop thread, that can't be spotted testing directly the client in a `@QuarkusTest` (because blocking the test main thread doesn't cause any error: it simply blocks).

Reply:
Added comment also in the code :+1: